pipelines/edx/mfe-frontend-app-learning-mitx-staging-qa.yaml

```diff
@@ -0,0 +1,76 @@
+---
+resource_types:
+- name: rclone
+  type: docker-image
+  source:
+    repository: mitodl/concourse-rclone-resource
+    tag: latest
+
+resources:
+- name: mfe-app-learning
+  type: git
+  source:
+    uri: https://github.com/edx/frontend-app-learning.git
+    branch: open-release/maple.master
+- name: mfe-app-learning-bucket
+  type: rclone
+  source:
+    config: |
+      [s3-remote]
+      type = s3
+      provider = AWS
+      env_auth = true
+      region = us-east-1
+
+jobs:
+- name: build
+  plan:
+  - get: mfe-app-learning
+    trigger: true
+  - task: npm-install-and-build
+    config:
+      platform: linux
+      image_resource:
+        type: registry-image
+        source:
+          repository: node
+          tag: 16-bullseye-slim
+      inputs:
+      - name: mfe-app-learning
+      outputs:
+      - name: mfe-app-learning/dist
+      params:
+        ACCESS_TOKEN_COOKIE_NAME: edx-jwt-cookie-header-payload
+        BASE_URL: https://staging-qa.mitx.mit.edu
+        CSRF_TOKEN_API_PATH: /csrf/api/v1/token
+        LMS_BASE_URL: https://staging-qa.mitx.mit.edu
```

Comment:
This is staying at mitx-qa-draft.mitx.mit.edu until we have the time to set up SAML for the other domain.